### ISSUE TITLE
new updates?

### ISSUE THREAD

**Issue by MrFuture0** [shaise/glMillSimulator#3]
@shaise do you accept new add-ons to this project? i have done some things (such as importing stl tools, moving tools simultaneously according to statically or dynamically printed nc data, replacing the panel being processed with a desired stl file ...)

**Comment by shaise** [comment]
Hi. 
No, no updates since this project was basically a preview i made. The real project with all the latest changes was merged into the freecad project (www.freecad.org)

**Comment by MrFuture0** [comment]
ah I misunderstood. I know you developed the project there, I made these codings, we can also update this project for solo developers, if you wish I can push you the codes, I coded all the improvements I said.

**Comment by shaise** [comment]
Great! please go ahead!

**Comment by MrFuture0** [comment]
there is problem like ( File uploads require push access to this repository. )

**Comment by shaise** [comment]
no, if you clone the repository, push to your clone, then create a pull-request that i can pull here
